lms/static/sass/course/wiki/_wiki.scss

```diff
@@ -422,7 +416,7 @@ section.wiki {
     }
 
     &.btn-primary {
-      @include button;
+      @include button(simple, $btn-default-color);
```

Comment:
New, darker button color. Note that this mixin is a borboun mixin, not the old edX one. I didn't think it was worth changing all the buttons to move away from borboun.